app/decorators/user_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class UserDecorator < Draper::Decorator
```

Comment:
You're right. I renamed it to `ParticipantDecorator` as not all users are anonymized and it is about participants: deb48aaf0b852344dbcea67008bfec3a34f146ef